renovate.default.json

```diff
@@ -0,0 +1,129 @@
+{
+  "enabled": true,
+  "dependencyDashboard": true,
+  "timezone": "Europe/Paris",
+  "schedule": [],
+  "rangeStrategy": "bump",
```

Comment:
bump the range even if the new version satisfies the existing range, e.g. ^1.0.0 -> ^1.1.0